grapher/axis/AxisViews.tsx

```diff
@@ -207,6 +220,9 @@ export class HorizontalAxisComponent extends React.Component<{
             />
         ) : undefined
 
+        const tickYPlacement = horizontalAxisLabelsOnTop
+            ? bounds.top - axis.tickFontSize / 2
+            : bounds.bottom - labelOffset
```

Comment:
Need to (re)move the tiny ticks too (see `{tickMarks}` being returned below). 
<img width="898" alt="Screenshot_2021-07-26_at_19_12_22" src="https://user-images.githubusercontent.com/1308115/127038015-f14f56c3-ce56-4cfd-9a50-e1cae349eaa4.png">